packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.tsx

```diff
@@ -150,7 +153,7 @@ const ConfirmSignupCode = ({
       if (isSyncDesktopIntegration(integration)) {
         // Connect another device tells Sync the user is signed in
         hardNavigateToContentServer(
-          `/connect_another_device${window.location.search}`
+          `/connect_another_device${location.search}`
```

Comment:
Was there a reason this was window.location... I can't recall now, but feel like maybe there was, so just double checking.

Reply:
I _think_ we just did this because we were doing it in other places and not always using the `useLocation` hook. I can't recall any reason to directly reference `window` but I'm also fine reverting if there may be a case to use it.